src/result/error_code.rs

```diff
@@ -13,6 +13,9 @@ use bindings::{
 pub struct HRESULT(pub u32);
 
 impl HRESULT {
+    pub const E_NOINTERFACE: Self = E_NOINTERFACE;
+    pub const E_POINTER: Self = E_POINTER;
```

Comment:
Marked this PR as draft because I'm not sure what the best place is to reexport these constants, without requiring every app to include `E_NOINTERFACE`/`E_POINTER` on their own regard.